### PR TITLE
Fix copy and add dark mode

### PR DESCRIPTION
**Pull request by esthermmoturi**
Things that have been implemented include:

- Style css files have been removed to ensure it is easier to keep up with changes and upgrades provided by hugo and docsy
- Dark mode was enabled for the CHT doc-site. This includes Auto, Dark and Light mode which can be changed from an icon close to the search box

Changes you might observe include:
- Colour changes for page-info, notes and other short-codes to accommodate dark mode.
-  The footer does not contain "All Rights Reserved" - this will be fixed if it is a necessity.
- All images are now placed below/before code snippets or text. This ~~fixes~~ [this issue](https://github.com/medic/cht-docs/issues/1475).
- Heading self-link(hover) from a chain to #.

### CODE REVIEW

**Comment by esthermmoturi** [comment]
Hey @jkuester , please let me know if you notice anything out of the ordinary with this change.

**Review by jkuester — CHANGES_REQUESTED** [review]
Dark mode AND -20,000 lines of CSS!  :heart_eyes: 

## Dark-mode

image

The embedded search results are not dark.  Maybe there is  a way to control this in our [gcse settings](https://www.docsy.dev/docs/adding-content/search/#setting-up-site-search)... :thinking: 

image

The code blocks are not dark.  Maybe we need to tweak the [chroma settings](https://www.docsy.dev/docs/adding-content/lookandfeel/#lightdark-color-themes)?

## Colors

I figure even if we switch to a dark theme, we still will want some custom colors (which is something with first-class support [in docsy](https://www.docsy.dev/docs/adding-content/lookandfeel/#site-colors). 

## Formatting

### Sidebar

Previously we had some custom CSS around the sidebar that was trimming up the indentation and padding so it was more efficient with its use of space.  IMHO the sidebar is one area that we might be okay keeping our customizations.  Even if future updates to Docsy break our sidebar layout, it should be easy to detect (since you see the same sidebar on every page) and simple(ish) to resolve (since our customizations are limited to some SCSS code in one file...). Ultimately, I am fine sticking with the default sidebar just for the sake of simplicity, but did want to mention that customizing it should be feasible too...

### Images

The image layout situation is much more tricky.  On one hand, it definitely simplifies things to just inline the images without being able to float them next to each other or to the text.  However, I do think it makes the docs quite a bit more clunky to read. :disappointed: It is a bit confusing to me that the image floating ever worked with us just putting the bootstrap classes in with the `figure` shortcode. Not really surprised that  that we are getting stuff like https://github.com/medic/cht-docs/issues/1475 now.  

I am not a web dev expert, but IMHO if we want do more complex layouts than just inlineing the images with basic markdown formatting, we should just commit to doing it with explicit bootstrap containers.  Docsy uses bootstrap and it seems like this has to be how things have been working in the past for the floating images.  Going forwards perhaps we need to be more explicit about the layout by specifying containers/rows/columns.  (Could write our own shortcodes to make this easier, or maybe look into [this Hugo Module](https://bootstrap.hugomods.com/docs/).)

**Comment by esthermmoturi** [comment]
Thank you @jkuester for the feedback.I'll add a checklist below for things that should be fixed.

- [ ] Fix code highlighting colour in dark mode
- [ ] Add sidebar customization

**Comment by esthermmoturi** [comment]
By adding the ` _variables_project.scss` file, I have retained the padding and colour customization of the sidebar. The only customization I have removed was for the increased height of the header navbar since the light/dark menu was misplaced. This also has undone the image repositioning thus issue https://github.com/medic/cht-docs/issues/1475 remains open.

File: assets/scss/_variables_project.scss [hunk 1]
```diff
@@ -80,7 +80,7 @@ h2, h3 {
   }
   .td-sidebar-link {
     &__page {
-      color: #222222;
+      // color: #222222;
```
Comment (jkuester) [inline assets/scss/_variables_project.scss:83]:
```suggestion
```

**Review by jkuester — APPROVED** [review]
:heart_eyes:  Looks great!  Just one minor suggestion to get the code syntax colors back.

> I think we can leave the search results as they are, do you think I should open a separate issue to fix it?

Agreed. Probably not a huge deal to fix this right now.  (My guess is that we will eventually have to target this with some custom CSS... :sweat: )  But, yeah, lets go ahead and log an issue for it just to track the fact that we know of the issue.

File: assets/scss/_variables_project.scss
[the remark this answers is not in the document]
Comment (esthermmoturi) [reply assets/scss/_variables_project.scss:1]: Thanks for being thorough and adding suggestions. I'll definitely reach out for any CSS queries or PRs

**Pull request merged**